public/locales/ja/common.json

```diff
@@ -46,12 +47,23 @@
   "QUESTS_INPUT_PLACEHOLDER": "キーワード",
   "QUESTS_SUBMIT_BUTTON": "キーワードを送信",
 
+  "CLAIM_HEADING": "$HENKAKUを受けとる",
+  "CLAIM_EXPLANATION_HEADING": "トークンを受け取りましょう。",
+  "CLAIM_EXPLANATION_BODY": "$HENKAKU受けとり可能です。",
+  "CLAIM_EXPLANATION_HEADING_NO_NFT": "まずはNFTを取得しましょう！",
+  "CLAIM_EXPLANATION_BODY_NO_NFT": "$HENKAKUを受け取るには家紋NFTが必要です。",
+  "CLAIM_CLAIMABLE_AMOUNT_0": "しかし、$HENKAKUを受け取れるほどのポイントをお持ちではありません。",
+  "CLAIM_CLAIMABLE_AMOUNT_1": "現在",
+  "CLAIM_CLAIMABLE_AMOUNT_2": "$HENKAKUを受けとることができます。",
+  "CLAIM_SUBMIT_BUTTON": "受けとる",
```

Comment:
can you change to like this structure ?

```json
{
"clam": {
  "header": "hoge",
  "body": "hoge"
}, 
"common": {
 "header": "foobar",
 ....
}
}
```

Reply:
That's a great idea! But I think we should apply this to the whole string files at once, so I made it a different issue: #84 

This PR is focussed on the Claim page.

Reply:
ok